css-view-transitions-1/Overview.bs

```diff
@@ -806,6 +814,9 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 						Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
 
+					1. If |transition|'s [=ViewTransition/initial snapshot viewport size=] is not equal to the [=snapshot viewport size=],
```

Comment:
Not sure if we need to specify exact when we do this check, or we can just add it to the step that's setting it:

```
1. Set |transition|'s [=ViewTransition/initial snapshot viewport size=] to the [=snapshot viewport size=]. If this size changes after this point, skip the transition.
```

I'm fine with this too.

Reply:
I thought about this, but doing it properly would be more complicated.

Changes to the snapshot viewport happen outside the page, so to do things on the page, it would need to queue a task. Because a task is queued, we can't guarantee the same view transition is active compared to when the task was queued, so that will need to be verified before skipping the transition.